db/schema.rb

```diff
@@ -66,11 +66,18 @@
     t.string "provider", default: "email", null: false
     t.string "uid", default: "", null: false
     t.string "encrypted_password", default: "", null: false
+    t.string "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string "unconfirmed_email"
```

Comment:
Should it be here? I think you have locally run a migration that isn't on the scope of this PR